content/boards/arduino-mega-2560/index.md

```diff
@@ -16,6 +16,17 @@ The Mega 2560 Rev3 is a classic Arduino board with a large number of IO pins and
 - 54 digital IO pins, 15 with PWM support.
 - 16 analog inputs (can be used as digital inputs).
 
+| Device | Limit | Notes |
+| ------------- | :-----------: | ----------- |
+| [Analog inputs](/devices/potentiometer/) | 16 | Potentiometers or linear hall sensors can be connected. |
```

Comment:
Table should be sorted alphabetically by device name. You can have VSCode do this for you, just select the rows, bring up the command palette, and search for the sort lines command.